contrib/completion/bash/docker

```diff
@@ -700,6 +700,7 @@ _docker_daemon() {
 		--ip-masq=false
 		--iptables=false
 		--ipv6
+		--raw-logs=true
```

Comment:
My understanding is the completions are for values you'd actually ever set (aka non-default). If you look a few lines up, the file says `--icc=false` and I believe `icc` is actually true by default.